gradio_app.py

```
@@ -0,0 +1,1258 @@
+    PITCH_SHIFT_MAX = 1.5
+    SPEED_CHANGE_MIN = 0.9
+    SPEED_CHANGE_MAX = 1.1
+
+# Audio Processing Class (same as before)
+class AudioProcessor:
+    def __init__(self, config=AudioConfig):
+        self.config = config
+
+    def load_audio(self, file_path):
+        try:
+            audio, sr = librosa.load(file_path, sr=self.config.SAMPLE_RATE)
```

Ruff (0.13.1)

Unpacked variable `sr` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)